app/controllers/imprints_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ImprintsController < ApplicationController
```

Comment:
Create module Imprint, put controller inside and rename it to PrintsController or whatever better.